staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -538,6 +547,8 @@ func (f *DeltaFIFO) Replace(list []interface{}, resourceVersion string) error {
 			if keys.Has(k) {
 				continue
 			}
+			// Delete pre-existing items not in the new list.
+			// This could happen if watch deletion event was missed.
```

Comment:
This function is called on a reconnection. Missing watch events is a serious bug and should never happen. Not getting events while not connected is expected and the whole point of this function is to reconcile things. :) Can you make this comment clearer so there's no chance of readers confusing those two things?

Reply:
Rephrased the comment to 
> This could happen if watch deletion event was missed while not connected.

Also rephrased the comment around DeletedFinalStateUnknown to align with the comments here.

Let me know what you think.